crates/roc_host/src/lib.rs

```diff
@@ -135,9 +134,9 @@ pub extern "C" fn main() -> i32 {
         return 1;
```

Comment:
This currently prints "Missing args..." if there are extra args as well, consider handling too many args and too few args separately.

Reply:
Done